### ISSUE TITLE
Babylon Debug Screen No Longer Works

### ISSUE THREAD

**Issue by PandawanFr** [andyhall/noa#90]
Hey,

After the move from Babylon's ES5 to ES6 modules (which separated babylon into a bunch of smaller modules to reduce the import size), the debug layer broke (the one that should display when pressing Z).

I looked it up and [it seems that importing `@babylon/inspector` and `@babylonjs/core/Debug/debugLayer` would fix the issue](https://doc.babylonjs.com/features/es6_support#almighty-inspector).

Perhaps using a dynamic import [when debug is enabled](https://github.com/andyhall/noa/blob/master/src/index.js#L222) could allow for the side effects to only be called when needed.

Thanks!

**Comment by andyhall** [comment]
Hi, thanks for the report!

It sounds to me like the answer here is to move the "press Z to debug" from the engine into the demo, so that the game client controls whether the module gets pulled in or not. I'll go ahead with that, but let me know if you disagree. Thanks!

**Comment by andyhall** [comment]
Hi, this is fixed - I added the imports and the Z shortcut to `noa-examples` test world, and they'll be removed from `noa` shortly.

By the way - when I tried this the scene inspector launched normally, but many of its features threw errors as soon as I tried to use them. I don't know why that is - I assume the inspector makes some assumptions about how the scene is put together? If you know any details let me know.
